python/torch_mlir/__init__.py

```diff
@@ -15,6 +15,9 @@
 from .compiler_utils import run_pipeline_with_repro_report
 from torch_mlir.dialects.torch.importer.jit_ir import ClassAnnotator, ImportOptions, ModuleBuilder
 
+# This value is taken from `ShapedType::kDynamicSize` variable
+# defined here: "mlir/IR/BuiltinTypeInterfaces.h"
+MLIR_DYNAMIC_STRIDE = -9223372036854775808
```

Comment:
It is really not okay to leak these internal implementation details into the Python API like this.